src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlottingPerSeg.R

```diff
@@ -0,0 +1,88 @@
+#NOTE: the Java wrapper for this script first sources CNV_plotting_library.R
```

Comment:
Because debugging in R is such a nightmare, I believe separating these scripts is worth the hassle of having duplicated code.